targets/targets.json

```diff
@@ -2640,4 +2640,22 @@
         "inherits": ["SARA_NBIOT"],
         "extra_labels": ["ublox", "HI2110", "SARA_NBIOT"]
     }
+    "RTL8195A": {
```

Comment:
The line above should have delimiter (``,``), this i believe is causing CI failures.